package.json

```diff
@@ -48,6 +48,9 @@
     "express": "4.9.5",
     "express-session": "1.10.4",
     "faker": "1.1.0",
+    "firebase": "3.5.1",
+    "firebase-api": "1.0.0",
+    "firebase-client": "0.1.1",
```

Comment:
These packages should be added into dependencies list in client/package.json file.